network/p2p/p2p.go

```diff
@@ -321,3 +343,55 @@ func formatPeerTelemetryInfoProtocolName(telemetryID string, telemetryInstance s
 		base32.StdEncoding.EncodeToString([]byte(telemetryInstance)),
 	)
 }
+
+var private6 = parseCIDR([]string{
```

Comment:
Are these needed because isPublic will be true for them?

Reply:
correct